src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -1432,7 +1432,6 @@ public override DynamicMetaObject Bind(DynamicMetaObject target, DynamicMetaObje
                     indexResult.Expression,
                     Expression.Catch(exception,
                         Expression.Block(
-                            Expression.Call(CachedReflectionInfo.CommandProcessorBase_CheckForSevereException, exception),
```

Comment:
Minor: the `Expression.Block` seems unnecessary now, shall we remove it as well? It can be:
```
Expression.Catch(exception, result);
```